.travis.yml

```diff
@@ -74,6 +66,18 @@ script:
   - coverage run manage.py test
   - echo 'travis_fold:end:django-tests'
 
+  # TODO: Start up solr (and neo4j?)
+  - echo 'travis_fold:start:cypress'
+  - python manage.py loaddata guest.json
+  - python manage.py activate_user guest
+  - python manage.py add_users_to_public_group
+  - python manage.py collectstatic --noinput -v 0
+  - python manage.py runserver --insecure &
```

Comment:
These five lines should probably be in the install and/or before_script sections. Ideally, Puppet should be used for provisioning to avoid code duplication.